definitions/checks/disk/available_space_var.rb

```diff
@@ -0,0 +1,26 @@
+module Checks
+  module Disk
+    class AvailableSpaceCandlepin < ForemanMaintain::Check
+      metadata do
+        label :available_space_cp
+        description 'Check to make sure /var/lib/candlepin has enough space'
+        tags :pre_upgrade
+      end
+
+      MAX_USAGE_IN_PERCENT = 90
+
+      def run
+        assert(enough_space?, "System has more than #{MAX_USAGE_IN_PERCENT}% space used"\
+              ' on /var/lib/candlepin')
+      end
+
+      def enough_space?
+        check_dir = '/var/lib/candlepin'
+        check_dir = '/var/lib' unless File.exist? check_dir
```

Comment:
Instead of checking defining the check_dir here and then confirming if it exists I think better way would be to add directory in candlepin feature ?
I have not seen anyone changing the default directory structure for candlepin, so in that case, we can just limit this check for feature(:candlepin) ?

Reply:
Please see updated implementation.